types/cache-interceptor.d.ts

```diff
@@ -31,10 +31,7 @@ declare namespace CacheHandler {
     path: string
   }
 
-  export interface GetResult {
-    response: CachedResponse
-    body?: Readable
-  }
+  type GetResult = CachedResponse & { body: null | Readable | Iterable<Buffer> | Buffer | Iterable<string> | string }
```

Comment:
Could you add some more testing in the interceptor test file for each of the body types?

Reply:
I'll change it back